### PR TITLE
servers: support externalTrafficPolicy in service

### PR DESCRIPTION
**Pull request by rcarpa**

### CODE REVIEW

File: charts/rucio-server/values.yaml [hunk 1]
```diff
@@ -184,6 +185,7 @@ wsgi:
   daemonProcesses: "4"
   daemonThreads: "4"
 
+## values used to configure apache
 httpd_config:
   mpm_mode: "event"
```
Comment (maany) [inline charts/rucio-server/values.yaml:190]: should this be in the flux config or do we want to use the event mode by default?

Comment (mlassnig) [reply charts/rucio-server/values.yaml:190]: mpm_event is the best mode that apache has, so can/should be default. there's no upsides to using mpm_worker or mpm_prefork

**Review by maany — APPROVED** [review]
LGTM! podAffinity config looks good.

File: charts/rucio-server/values.yaml (hunk 1 above)
Comment (rcarpa) [reply charts/rucio-server/values.yaml:190]: you are right, I didn't realize that removing the duplicate key (httpd_config) in the yaml below will result in a change of behavior.

**Pull request merged**